src/OlStyleParser.ts

```diff
@@ -315,6 +315,26 @@ class OlStyleParser implements StyleParser {
     });
   }
 
+  /**
+   * Get the OL Style object  from an GeoStyler-Style CircleSymbolizer.
+   *
+   * @param {IconSymbolizer} symbolizer  A GeoStyler-Style IconSymbolizer.
+   * @return {object} The OL Style object
+   */
+  getOlIconSymbolizerFromIconSymbolizer(symbolizer: IconSymbolizer) {
+    return new ol.style.Style({
+      image: new ol.style.Icon({
+        src: symbolizer.image,
+        crossOrigin: 'anonymous',
+        offset: symbolizer.offset,
```

Comment:
Since offset isn't tested: Do we have the same format in our internal style as OpenLayers? For labels they differed (array vs. 2 properties for x and y)

Reply:
Offset acts very different in OpenLayers as i would expect it. Thats why i didn't add it to the tests. I just forgot to remove it here and will do this know...